tests/client/amo/reducers/testLanding.js

```diff
@@ -52,11 +52,11 @@ describe('landing reducer', () => {
         addonType, featured, highlyRated, loading, popular,
       } = landing(initialState, getLanding({ addonType: ADDON_TYPE_THEME }));
 
-      assert.equal(addonType, ADDON_TYPE_THEME);
-      assert.equal(loading, true);
-      assert.deepEqual(featured, { foo: 'bar' });
-      assert.deepEqual(highlyRated, { count: 0 });
-      assert.deepEqual(popular, { results: [] });
+      expect(addonType).toEqual(ADDON_TYPE_THEME);
+      expect(loading).toEqual(true);
```

Comment:
This should be `.toBe(true)` to get strict equality, right? I think there are a couple more in here which might be worth hunting down.